### ISSUE TITLE
Bypass login for local network

### ISSUE THREAD

**Issue by Spamme1** [hadouken/hadouken#218]
It would be nice if the login (basic authentication) can be bypassed if accessed from the local network (home network). Some torrent clients offer an option to bypass the login when accessed from the localhost, I find it interesting but not really usefull, because when you run a torrent client as a service, it is obvious that the client run on a server and you access it from another computer in the same network.

**Comment by vktr** [comment]
I'll add the feature tag although I'm not sure if this is a feature I want to implement - the local network (10.0.0.0/8, 172.16.0.0/12, and 192.168.0.0/16) is quite big and by just allowing anything from there to Hadouken could be a security risk.

Most browsers have an option to remember credentials which would already work fine with Hadouken.

**Comment by kitchung** [comment]
Maybe an adjustable subnet setting, ie for my personal network, i may want
to scope it to 10.1.1.0/24.

**Comment by Spamme1** [comment]
@vktr you are right although I configure always the browser to never remember history, passwords and to delete every file when closing.
@kitchung yes a subnet mask would be good idea

**Comment by vktr** [comment]
Good suggestions :) I'll see what I can do!
